frameit/lib/frameit/editor.rb

```diff
@@ -233,27 +231,20 @@ def put_title_into_background_stacked(background, title, keyword)
       resize_text(title)
       resize_text(keyword)
 
-      title_width = title.width
-      keyword_width = keyword.width
-
-      vertical_padding = vertical_frame_padding
```

Comment:
undo

Reply:
I expect you only refer to `vertical_padding = vertical_frame_padding`, not the `...width` parameters above?

Reply:
yep, only that exact line - no idea why Github shows those as well...

Reply:
Yeah, I added the comment to explain why using it. And to identify it for #11648